conf/messages

```diff
@@ -484,7 +484,7 @@ page.title.change.sample.return=Should this sample be returned?
 page.title.reject.case=Change case status to: Rejected
 page.title.case.rejected=This case has been rejected
 case.rejected.header = The {0} case has been rejected
-page.title.release.case=Choose a team to release this case to
+page.title.release.case=Choose a team to release this case to queue
```

Comment:
should it say "queue"?

Reply:
@oliwiaKot thanks for flagging above and done the correction.